rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/PolicyConfigResource.java

```diff
@@ -66,7 +66,7 @@
             ((BrooklynObjectInternal)policy).config().getInternalConfigMap().getAllConfigInheritedRawValuesIgnoringErrors() ).getAllConfig();
         Map<String, Object> result = Maps.newLinkedHashMap();
         for (Map.Entry<String, Object> ek : source.entrySet()) {
-            result.put(ek.getKey(), getStringValueForDisplay(brooklyn(), policy, ek.getValue()));
+            result.put(ek.getKey(), WebResourceUtils.getValueForDisplay(mapper(), getStringValueForDisplay(brooklyn(), policy, ek.getValue()), true, true));
```

Comment:
This is not needed, fixes only needed for methods returning `String`. This one will get converted to json by the framework.